index.html

```diff
@@ -71,6 +72,110 @@ <h2>Software Developer</h2>
         </div>
 
     </footer>
+    <section class="work-sect">
+        <div>
+            <h2>MY RECENT WORK</h2>
+        </div>
+        <div class="work-sect">
+            <img class="card-image" src="/assets/images/dummy-image-24toolbox-350x450.png" alt="dummy image">
+            <h3>Multi-Post Stories</h3>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's
+                standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+
+            <ul class="grid-container">
+                <li>Css</li>
+                <li class="border1">Html</li>
+                <li class="border2">Bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+
+
+        </div>
+
+    </section>
+    <section class="work-sect">
+        <div>
+            <h2>MY RECENT WORK</h2>
+        </div>
+        <div class="work-sect">
+            <img class="card-image" src="/assets/images/dummy-image-24toolbox-350x450.png" alt="dummy image">
+            <h3>Multi-Post Stories</h3>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's
+                standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+
+            <ul class="grid-container">
+                <li>Css</li>
+                <li class="border1">Html</li>
+                <li class="border2">Bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+
+
+        </div>
+
+    </section>
+    <section class="work-sect">
+        <div>
+            <h2>MY RECENT WORK</h2>
+        </div>
+        <div class="work-sect">
+            <img class="card-image" src="/assets/images/dummy-image-24toolbox-350x450.png" alt="dummy image">
+            <h3>Multi-Post Stories</h3>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's
+                standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+
+            <ul class="grid-container">
+                <li>Css</li>
+                <li class="border1">Html</li>
+                <li class="border2">Bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+
+
+        </div>
+
+    </section>
```

Comment:
- [ ] For each project, ensure the section uses the font and colors background and font) defined for the template. Also put the margins (top, bottom, left, right) defined in the design into consideration.
Your Project
![Screenshot from 2021-09-17 02-21-13](https://user-images.githubusercontent.com/65501535/133698690-95e3de8c-e4a9-4b53-ab2e-285e494ef7ab.png)

Original design
![Screenshot from 2021-09-17 02-23-58](https://user-images.githubusercontent.com/65501535/133699133-434e4ef8-3969-4167-a708-55cc9d98f12c.png)